index.html

```diff
@@ -2974,134 +2983,367 @@ <h3>Obtaining a manifest</h3>
+			            <code>undefined</code>, add<br />
+			            <code class="json">"inLanguage": lang</code><br /> to <var>manifest</var>
+						<details>
+			                <summary>Explanation</summary>
+			                <p>This step ensures that the language explicitly set in the primary entry page is valid for an embedded manifest content. I.e.,</p>
+			                <pre class="example">
+&lt;html&gt;
+&lt;head&gt;
+    &#8230;
+    &lt;script type="application/ld+json" lang=ur&gt;
+    {
+        "@context" : ["https://schema.org","https://www.w3.org/ns/wp-context"],
+        &#8230;
+    }
+    &lt;/script&gt;
+			                </pre>
+			                <p>yields (unless the language and the direction are set explicitly in the manifest):</p>
+			                <pre class="example">
+{
+    "@context"    : ["https://schema.org","https://www.w3.org/ns/wp-context"],
+    "inLanguage"  : "ur",
+    &#8230;
+}
+			                </pre>
+			        </details>
+			        </li>
+			        <li id="can-set-dir"> (<a href="#language-and-dir"></a>) if <var>manifest["inDirection"]</var> is
+			                <code>undefined</code> and the value of <var>dir</var> is <em>not</em>
+			            <code>undefined</code>, add<br />
+			            <code class="json">"inDirection": dir</code><br /> to <var>manifest</var>
+			            <details>
+			                <summary>Explanation</summary>
+			                <p>This step ensures that the base direction explicitly set in the primary entry page is valid for an embedded manifest content. I.e.,</p>
+			                <pre class="example">
+&lt;html&gt;
+&lt;head&gt;
+    &#8230;
+    &lt;script type="application/ld+json" dir=rtl lang=ur&gt;
+    {
+        "@context" : ["https://schema.org","https://www.w3.org/ns/wp-context"],
+        "inLanguage"  : "ur",
+        &#8230;
+    }
+    &lt;/script&gt;
+			                </pre>
+			                <p>yields (unless the language and the direction are set explicitly in the manifest):</p>
+			                <pre class="example">
+{
+    "@context"    : ["https://schema.org","https://www.w3.org/ns/wp-context"],
+    "inLanguage"  : "ur",
+    "inDirection" : "rtl"
+    &#8230;
+}
+			                </pre>
+			        </details>
+			        </li>
+			        <li> (<a href="#default-reading-order"></a>) if <var>manifest["readingOrder"]</var> is
+			                <code>undefined</code>, let <var>u</var> be the value of <a
+			                data-cite="!dom#concept-document-url">document.URL</a>, and add<br />
+			            <code class="json">"readingOrder": [{"type": ["LinkedResource"], "url": u}]</code><br /> to the
+			                <var>manifest</var>
+						<details>
+							<summary>Explanation</summary>
+							<p>
+								In case the publication consists of the primary entry page as an only content, the default reading order can be ommited; it will consist, automatically, of that single resource.
```

Comment:
"an only content" probably s/b "the only", but maybe rephrase this more simply as "If the publication consists only of the primary entry page"

Reply:
Also, s/ommited/omitted/